dist/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Oga-Dela Art Gallery</title>
+  <script defer src="main.js"></script></head>
+
+  <body>
+    <header>
+      <div id="logo">
+        <a href="#"> <img id="logo1" src="#" alt="logo-image"></a>
+        <h1 id="title">Oga-Dela Arts</h1>
+      </div>
+      <nav id="header-nav">
+        <ul id="nav-list">
+          <Li class="list">Paintings (<span class="paint-counter">0</span>)</Li>
+          <Li class="list">Artist</Li>
+          <Li class="list">Category</Li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section id="art-container" class="items">
+
+      </section>
+    </main>
+
+    <footer>
+      <p class="footer-section"> &copy; Created By Ogaga && Mandela Under Creative Commons Licence </p>
+    </footer>
+
+    <div class="hidden bg-popup">
+      <div id="popup" class="popup">
+        <span id="closeIcon">X</span>
+        <div class="image-container">
+          <img src="#" alt="image" id="popup-image">
+        </div>
+        <div class="details-container">
+        </div>
+
+        <span id="comment-count"></span>
+        <div>
+          <h2 class="comments-counter"></h2>
+          <div class="comments-container"></div>
+        </div>
```

Comment:
- [ ] Card details should follow the mockup as well. 

Yours  | Mockup
------------- | -------------
<img width="438" alt="Screenshot 2022-05-20 at 18 11 33" src="https://user-images.githubusercontent.com/77038610/169546325-b573f419-2614-4317-b14c-a74aa340870b.png">  | <img width="150" alt="Screenshot 2022-05-20 at 18 10 33" src="https://user-images.githubusercontent.com/77038610/169546137-814a9b30-979c-4dd1-80c6-eee7f815a17a.png">

Kindly align the content as provided in the mock up and make the comments button look like button, add cursor pointer to make it clear that it is a button. 

- [ ] Kindly make sure that all cards are aligned the same way, currently some  titles take more lines and as a result push likes button and comments button out of visual border:
<img width="1382" alt="Screenshot 2022-05-20 at 18 15 11" src="https://user-images.githubusercontent.com/77038610/169547059-4c847bfa-bf57-42d7-95f3-ad3e28705292.png">

- [ ] OPTIONAL: This project can be a very good showcase for your future employment, so let's try to improve the UI. There are several ways you can improve the UI:
- choose different text color with higher contrast, I like the color of the text, but as a user I feel that I need more contrast, try to make it brighter. 
- Decrease the size of images, It feels like they are too massive. 
- Add more space, more margins and paddings 
etc,. just have fun guys! But this requirement is optional...